docs/DeveloperGuide.adoc

```diff
@@ -96,11 +96,16 @@ The `UI` component,
 image::LogicClassDiagram.png[]
```

Comment:
Maybe can modify the diagram above so that these 2 diagrams don't repeat each other??